randovania/games/dread/item_database/item-database.json

```diff
@@ -709,6 +709,42 @@
             "probability_multiplier": 1,
             "preferred_location_category": "minor",
             "extra": {}
+        },
+        "Flash Shift Upgrade": {
+            "item_category": "aeion",
+            "broad_category": "aeion",
+            "model_name": "item_flashshiftupgrade",
+            "progression": [
+                "FlashUpgrade"
+            ],
+            "default_shuffled_count": 0,
+            "default_starting_count": 2,
+            "ammo": [],
+            "unlocks_ammo": false,
+            "hide_from_gui": false,
+            "must_be_starting": false,
+            "probability_offset": 0,
+            "probability_multiplier": 1,
+            "preferred_location_category": "major",
+            "extra": {}
+        },
+        "Speed Booster Upgrade": {
+            "item_category": "misc",
+            "broad_category": "misc",
+            "model_name": "item_speedboostupgrade",
+            "progression": [
+                "SpeedBoostUpgrade"
+            ],
+            "default_shuffled_count": 0,
+            "default_starting_count": 0,
+            "ammo": [],
+            "unlocks_ammo": false,
+            "hide_from_gui": false,
+            "must_be_starting": false,
+            "probability_offset": 0,
+            "probability_multiplier": 1,
+            "preferred_location_category": "major",
+            "extra": {}
```

Comment:
please add a `warning` field to each of these (which is admittedly somewhat poorly named) with the description of each one. for speed upgrade, please mention the minimum possible charge time of 0.25s as well